Sources/Fluent/Entity/SoftDeletable.swift

```diff
@@ -91,6 +91,12 @@ extension QueryRepresentable where E: SoftDeletable, Self: ExecutorRepresentable
         query.includeSoftDeleted = true
         return query
     }
+
+    public func forceDelete() throws -> Query<E> {
```

Comment:
This function should not return anything and should delete the entities.

You should just be able to call `try query.raw()` after setting the action to `.delete`.